### ISSUE TITLE
Use Perceval summary

### ISSUE THREAD

**Issue by sduenas** [chaoss/grimoirelab-kingarthur#75]
Since Perceval incorporates a summary object with the result of a fetching process (see chaoss/grimoirelab-perceval/issues/529), Arthur doesn't need to generate this summary by its own.

`JobResult` and `PercevalJob` classes should be rewritten to add the information that comes from Perceval.

**Comment by sduenas** [comment]
While I was implementing this feature I had big problems trying to make it work with the system that resumes jobs that have failed. The main problem is **resuming** requires to use the old summary to rerun a job when it fails. It is complicated to keep all that data and it looks like an antipattern to me.

When we developed resuming feature we decided to integrate it inside of `execute_perceval_job` because it was the only place with enough information to resume a job.

Since Perceval implemented a retry system and it produces a summary with the status of the fetch process, this might no longer needed. The scheduler will be able to reschedule a new job if necessary.

We need to refactorize that code before continuing with this task.
